packages/app/.env

```diff
@@ -12,6 +12,7 @@
 # Find your chainID here: https://chainlist.org/
 # By default we use 5, or the Goerli network
 CHAIN_ID=5
+NEXT_PUBLIC_CHAIN_ID=5
```

Comment:
Might be worth moving `CHAIN_ID` to `NEXT_PUBLIC_CHAIN_ID` rather than duplicating.